### ISSUE TITLE
[feature-request] GUI First time wizard : please pop up dialog to encrypt

### ISSUE THREAD

**Issue by Technologov** [dashpay/dash#1379]
When a user first install a Dash Core on their PC, we should pop up a first time dialog and recommended to encrypt a wallet.

Else a backup file can be stolen.

A tale:
Once upon a time I put my Bitcoin wallet on my web server, in a secret folder, that I thought was inaccessible, yet a hacker somehow got access to it. What saved me at the time was very long password, that he was unable to crack. He was able to see my balance, though, which is bad enough.
Without an encryption, he would have stolen all of my Bitcoins.

-Technologov

**Comment by schinzelh** [comment]
I agree, we should "advertise" encryption more, or even go to "encryption = default" mode.

Plus am not happy about our autobackup function keeping unencrypted copies of wallet.dat ...

**Comment by akijuh** [comment]
I agree, encrypt popup during first launch would be nice to have.

Also **optional pincode input popup** before wallet shows up.
Now wallet just start and shows all, so anybody can spy on your dash stuff.

**Comment by forro68** [comment]
Is there any chance this could be included in 0.17?
